pkg/products/threescale/reconciler.go

```diff
@@ -3062,121 +3066,6 @@ func (r *Reconciler) deleteConsoleLink(ctx context.Context, serverClient k8sclie
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
-func (r *Reconciler) reconcileDeploymentConfigs(ctx context.Context, serverClient k8sclient.Client, productNamespace string) (integreatlyv1alpha1.StatusPhase, error) {
-
-	for _, name := range threeScaleDeploymentConfigs {
-		deploymentConfig := &appsv1.DeploymentConfig{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      name,
-				Namespace: productNamespace,
-			},
-		}
-
-		podPriorityMutation := resources.MutatePodPriority(r.installation.Spec.PriorityClassName)
-
-		phase, err := resources.UpdatePodTemplateIfExists(
-			ctx,
-			serverClient,
-			resources.SelectFromDeploymentConfig,
-			resources.AllMutationsOf(
-				resources.MutateZoneTopologySpreadConstraints("app"),
```

Comment:
We probably still need `resources.MutateZoneTopologySpreadConstraints("app")` for zone topology spread constaints for multi az unless we can also define this via the ApiManager CR on 0.11.6-mas 🤔

Reply:
yes. This is still in progress